doc/packer.txt

```diff
@@ -44,6 +44,7 @@ FEATURES                                         *packer-intro-features*
 - Uses jobs for async installation
 - Support for `git` tags, branches, revisions, submodules
 - Support for local plugins
+- Support snapshots for plugins
```

Comment:
Nitpick: would prefer "Support for saving/restoring snapshots of plugin versions" (or similar, just something a bit more descriptive).